Analysis/include/SummaryTuple.h

```diff
@@ -50,9 +55,15 @@ INITIALIZE_TREE(ntuple, SummaryTuple, SUMMARY_DATA)
     VAR(Float_t, genEventWeight) /* gen event weight */ \
     VAR(Float_t, gen_top_pt) /* pt of gen ME top */ \
     VAR(Float_t, gen_topBar_pt) /* pt of gen ME anti-top */ \
+    VAR(Int_t,   genEventType) /* top gen event type */ \
     VAR(Float_t, lhe_H_m) /* mass of lhe H */ \
     VAR(Float_t, lhe_hh_m) /* mass of lhe hh pair */ \
     VAR(Float_t, lhe_hh_cosTheta) /* cos(theta) between h and z-axis in the hh reference frame */ \
+    /* MC truth event splitting */ \
+    VAR(std::vector<UInt_t>, lhe_n_partons) \
+    VAR(std::vector<UInt_t>, lhe_n_b_partons) \
+    VAR(std::vector<UInt_t>, lhe_ht10_bin) \
+    VAR(std::vector<ULong64_t>, lhe_n_events) \
```

Comment:
Branch lhe_n_events is not needed. Other 3 new branches should be UInt_t and not the vectors.